apps/src/pythonlab/layout/HorizontalLayout.tsx

```diff
@@ -15,11 +16,14 @@ const MIN_OUTPUT_HEIGHT = 120;
 const MIN_EDITOR_HEIGHT = 200;
 const INITIAL_INFO_PANEL_WIDTH = 300;
 const INITIAL_OUTPUT_HEIGHT = 300;
+const INITIAL_OUTPUT_HEIGHT_WIDGET = 800;
 const PROJECT_FOOTER_HEIGHT = 56;
 
 const HorizontalLayout: React.FunctionComponent<LayoutProps> = ({
   isProjectLevel,
 }) => {
+  const {levelProperties} = useCodebridgeContext();
+  const isWidgetView = !!levelProperties.widgetView;
```

Comment:
nit/observation - looks like we pass in `isProjectLevel` as a prop which I believed is derived from `levelProperties`, but we're getting `isWidgetView` from level properties via codebridge context. Might be nice to do one or the other?

Reply:
Nice observation. I think I like the idea of passing as a prop so updated `layoutProps` and added optional `isWidgetView`. Thanks!